gossip/config.go

```diff
@@ -219,6 +225,10 @@ func DefaultConfig(scale cachescale.Func) Config {
 		RPCGasCap:   50000000,
 		RPCTxFeeCap: 100, // 100 FTM
 		RPCTimeout:  5 * time.Second,
+
+		BatchRequestLimit: 1000,
+		// Limit 0 means, that there is no limit
+		JSTracerLimit: 0,
```

Comment:
Should there be a limit by default?

Reply:
After discussion with @jmpike changed limit to 1000